local.c

```diff
@@ -273,21 +266,20 @@ static int get_rel_timeout_ms(struct timespec *start, unsigned int timeout_rel)
 	if (timeout_rel > INT_MAX)
 		return INT_MAX;
 
-	return (int) timeout_rel;
+	return (int)timeout_rel;
 }
 
 static int device_check_ready(const struct iio_device *dev, short events,
-	struct timespec *start)
-{
-	struct pollfd pollfd[2] = {
-		{
-			.fd = dev->pdata->fd,
-			.events = events,
-		}, {
-			.fd = dev->pdata->cancel_fd,
-			.events = POLLIN,
-		}
-	};
+			      struct timespec *start)
+{
+	struct pollfd pollfd[2] = { {
+						    .fd = dev->pdata->fd,
+						    .events = events,
+				    },
+				    {
+						    .fd = dev->pdata->cancel_fd,
```

Comment:
Hmm - Placing Braces - I don't think this is proper Linux coding style. In fact you can find this or the other upstream.
But my preference is to put the opening brace last on the line '}, {'. This is more in sync with 
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces

Reply:
yeah, this looks funky indented;

Reply:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

have a look at the BraceWrapping options, I think you are talking about:
AfterStruct : false
SplitEmptyRecord : false